plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -280,6 +281,19 @@ public DeltaLakeConfig setVacuumMinRetention(Duration vacuumMinRetention)
         return this;
     }
 
+    public boolean isVacuumLoggingEnabled()
+    {
+        return vacuumLoggingEnabled;
+    }
+
+    @Config("delta.vacuum.logging.enabled")
```

Comment:
Logging can be misleading name in comparison with normal logs. Maybe `delta.vacuum.transactionlogging.enabled`? @ebyhr WDYT?

Reply:
Let's rename to `delta.vacuum.transaction-log.enabled`